### PR TITLE
Allow -Xmx to be specified as an argument to vespa-visit

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: vespaclient-java/src/main/sh/vespa-visit.sh [hunk 1]
```diff
@@ -74,16 +74,30 @@ findhost
...
+fi
+
 if [ "${VESPA_LOG_LEVEL}" = "" ]; then
     export VESPA_LOG_LEVEL=error,warning
 fi
 
-export MALLOC_ARENA_MAX=1 #Does not need fast allocation
-exec java \
+export MALLOC_ARENA_MAX=1 # Does not need fast allocation
+echo exec java \
```
Comment (arnej27959) [inline vespaclient-java/src/main/sh/vespa-visit.sh:96]: looks like you forgot to remove echo here

Comment (bratseth) [reply vespaclient-java/src/main/sh/vespa-visit.sh]: Thanks, fixed.

**Review by arnej27959 — APPROVED** [review]
taking a new look I saw some accidentally duplicated lines, so I pushed a commit with slight simplification.

**Comment by bratseth** [comment]
Great!

**Pull request merged**